modules/telco-ran-crs-cluster-tuning.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-crs.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cluster-tuning-crs_{context}"]
+= Cluster tuning reference CRs
+
+.Cluster tuning CRs
+[cols="4*", options="header", format=csv]
+|====
+Component,Reference CR,Optional,New in this release
+Cluster capabilities,xref:../../telco_ref_design_specs/ran/telco-ran-ref-du-crs.adoc#ztp-example-sno-yaml[example-sno.yaml],No,No
```

Comment:
This is the full SiteConfig. Maybe add a bit here to point out that we are calling out the `installConfigOverrides: ` config within that file?

Reply:
Added a note to the Cluster tuning section. Would prefer to not change this section since the table is generated by script.